integration/sbom_test.go

```diff
@@ -150,13 +150,28 @@ func TestSBOM(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "license check cyclonedx json",
+			args: args{
+				input:        "testdata/fixtures/sbom/license-cyclonedx.json",
+				format:       "json",
+				artifactType: "spdx",
```

Comment:
It should be `cyclonedx`, right?
```suggestion
				artifactType: ftypes.ArtifactType("cyclonedx"),
```